feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionsFragment.kt

```diff
@@ -113,6 +113,12 @@ class SessionsFragment : DaggerFragment() {
         binding.sessionsSheet.doOnApplyWindowInsets { _, insets, _ ->
             sessionSheetBehavior.peekHeight =
                 insets.systemWindowInsetBottom + initialPeekHeight + gestureNavigationBottomSpace
+            if (sessionSheetBehavior.state == BottomSheetBehavior.STATE_COLLAPSED)
```

Comment:
Thanks! 👍 
Can you add a comment? about this workaround 🙏

Reply:
Comment added 👍